### DISCUSSION TITLE
Issues with dimmer commands not registering

### DISCUSSION THREAD

**Discussion by haritrivedi** [iklein99/homebridge-smartthings#150] [General]
### I have zooz dimmer switches connected through smartthings and then through this plugin to homebridge. My problem is that my dimmer commands only register intermittently. Other times the light flickers to the new brightness and then reverts to the previous state. The only way to get it 'unstuck' is to go directly into ST and select a new brightness. Any ideas? 

These were working fine before inside ST and inside Google Home

**Comment by iklein99** [comment]
Are you using the webhook?  If you are polling, using a switch or dimmer like that will be problematic because the plugin polls each device no more than 4 seconds.  The polling doesn't work well with those types of things.

**Comment by haritrivedi** [comment]
Thanks for getting back - I am just using the plugin directly in homebridge and then exposing it to homekit. Its interesting because I have multiple of the same switches (25+) and it only happens on some of them. Do I need to check the device types or drivers in the IDE?

Thank you
